target/linux/generic/backport-5.15/850-v5.17-0011-PCI-aardvark-Disable-common-PHY-when-unbinding-drive.patch

```diff
@@ -18,9 +18,9 @@ Signed-off-by: Lorenzo Pieralisi <lorenzo.pieralisi@arm.com>
 
 --- a/drivers/pci/controller/pci-aardvark.c
 +++ b/drivers/pci/controller/pci-aardvark.c
-@@ -1734,6 +1734,9 @@ static int advk_pcie_remove(struct platf
- 	for (i = 0; i < OB_WIN_COUNT; i++)
- 		advk_pcie_disable_ob_win(pcie, i);
+@@ -1640,6 +1640,9 @@ static int advk_pcie_enable_phy(struct a
```

Comment:
Please remove this patch, it was applied upstream, see https://github.com/gregkh/linux/commit/ac76c36aa417014b4e46d4d1dee7b15c39ccbf8c